bitnami/orangehrm/templates/NOTES.txt

```diff
@@ -41,16 +52,36 @@
 This deployment will be incomplete until you configure OrangeHRM with a resolvable database
 host. To configure OrangeHRM to use and external database host:
 
-
 1. Complete your OrangeHRM deployment by running:
 
   helm upgrade {{ .Release.Name }} --set service.type={{ .Values.service.type }},mariadb.enabled=false,externalDatabase.host=YOUR_EXTERNAL_DATABASE_HOST bitnami/orangehrm
 
 {{- end }}
 
-{{- if and (contains "bitnami/" .Values.image.repository) (not (.Values.image.tag | toString | regexFind "-r\\d+$|sha256:")) }}
+{{- include "common.warnings.rollingTag" .Values.image }}
+{{- include "common.warnings.rollingTag" .Values.metrics.image }}
```

Comment:
Should we include this warning for `volumePermissions.image` and `certificates.image`? If so, we could move this to a small helper to avoid adding too many lines here.

Reply:
`volumePermissions` and `certificates` are using rolling tags (`buster`), we shouldn't be adding the warning for those, right?